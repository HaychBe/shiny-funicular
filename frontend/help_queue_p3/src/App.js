import Main from "./Components/HomePage/Main";
import Banner from "./Components/HomePage/Banner";

function App() {
  return (
    <body id="testBody">

        <Banner/>
        <Main/>

    </body>
  );
}

export default App;
