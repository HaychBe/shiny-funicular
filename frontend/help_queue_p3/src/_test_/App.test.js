// import { render, screen } from '@testing-library/react';
import App from '../App';
import '@testing-library/jest-dom';
import {create} from 'react-test-renderer';

describe(`Snapshot test of App`, () => {

  test(` App should match the snapshot every time it is rendered`, () => {
    const instance = create(<App/>).toJSON(); 
    expect(instance).toMatchSnapshot();
  });
});