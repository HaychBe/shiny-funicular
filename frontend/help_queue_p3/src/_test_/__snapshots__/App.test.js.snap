// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test of App  App should match the snapshot every time it is rendered 1`] = `
Array [
  <h1>
    NAV OR BANNER HERE
  </h1>,
  <p>
    Please wait, we're retrieving the data...
  </p>,
  <div
    className="spinner-grow text-primary"
  />,
]
`;
